openmetadata-ui/src/main/resources/ui/src/pages/teams/ImportTeamsPage/ImportTeamsPage.test.tsx

```diff
@@ -146,4 +181,81 @@ describe('ImportTeamsPage', () => {
 
     expect(await screen.findByText('TeamImportResult')).toBeInTheDocument();
   });
+
+  it('If team type is group and import type is team, Error placeholder should visible', async () => {
+    (getTeamByName as jest.Mock).mockImplementationOnce(() =>
+      Promise.resolve(MOCK_MARKETING_TEAM)
+    );
+    act(() => {
+      render(<ImportTeamsPage />);
+    });
+
+    expect(await screen.findByText('ErrorPlaceHolder')).toBeInTheDocument();
+    expect(
+      await screen.findByText(
+        'message.group-type-team-not-allowed-to-have-sub-team'
+      )
+    ).toBeInTheDocument();
+  });
+
+  it('On click of cancel, redirect to team tab of teams page', async () => {
+    act(() => {
+      render(<ImportTeamsPage />);
+    });
+    const cancelBtn = await screen.findByTestId('cancel');
+
+    await act(async () => {
+      fireEvent.click(cancelBtn);
+    });
+
+    expect(mockHistory.push).toHaveBeenCalledWith({
+      pathname: getTeamsWithFqnPath(MOCK_CURRENT_TEAM.fullyQualifiedName),
+      search: 'activeTab=teams',
+    });
+  });
+
+  // keep user import related test below this
+  it('UserImportResult should visible', async () => {
+    mockLocation.search = '?type=users';
+    (importUserInTeam as jest.Mock).mockImplementationOnce(() =>
+      Promise.resolve({
+        data: {
+          dryRun: true,
+          status: 'success',
+          numberOfRowsProcessed: 1,
+          numberOfRowsPassed: 1,
+          numberOfRowsFailed: 0,
+          importResultsCsv:
+            // eslint-disable-next-line max-len
+            'status,details,name*,displayName,description,email*,timezone,isAdmin,teams*,Roles\r\nsuccess,Entity updated,aaron_johnson0,Aaron Johnson,,aaron_johnson0@gmail.com,,false,Applications,DataSteward\r\n',
+        },
+      })
+    );
+    act(() => {
+      render(<ImportTeamsPage />);
+    });
+    const importBtn = await screen.findByTestId('import');
+    await act(async () => {
+      fireEvent.click(importBtn);
+    });
+
+    expect(await screen.findByText('UserImportResult')).toBeInTheDocument();
+  });
+
+  it('On click of cancel, redirect to users tab of teams page', async () => {
```

Comment:
Let's follow this template for tests cases

`should <Result> on <Action>`

If I rewrite above:

`should redirects to users tab to teams page on clicking on cancel`

Reply:
Thanks, ill keep it in mind.